### PR TITLE
the input-output parameters for the function distribute_local_to_glob…

### PR DESCRIPTION
**Pull request by Arezou-gh**
…al has been identified

### CODE REVIEW

File: include/deal.II/lac/constraint_matrix.h [hunk 1]
```diff
@@ -716,6 +716,8 @@ class ConstraintMatrix : public Subscriptor
    * index at the same time. This needs to be made sure from the caller's
    * site. There is no locking mechanism inside this method to prevent data
    * races.
+   * @param[in] local_vector, local_dof_indices, local_matrix are  input parameters.
+   * @return[out] global_vector  is an output parameter.
```
Comment (bangerth) [inline include/deal.II/lac/constraint_matrix.h]: You can't list multiple parameters in one `@param` description. Also, the `[in]` and `[out]` markers also tell that something is an input or output argument. What should be listed here should be what each one of these arguments actually represents.

**Comment by bangerth** [comment]
@Arezou-gh -- next time we meet, bring your laptop and I'll show you how to update this pull request.

**Comment by Arezou-gh** [comment]
I have updated according to the above comments. This is ready for review.

**Comment by bangerth** [comment]
May I interest someone to look at this first-timer's pull request? She's a student of mine, so I would appreciate someone else's eyes.

**Comment by kronbichler** [comment]
Nice. Thanks!

**Pull request merged**